src/main/java/com/Leapwork/Leapwork_plugin/PluginHandler.java

```diff
@@ -658,7 +672,7 @@ public RunItem getRunItem(AsyncHttpClient client, String controllerApiHttpAddres
                 else
                 {
                     Failure keyframes = getRunItemKeyFrames(client,controllerApiHttpAddress,accessKey,runItemId,runItem,scheduleTitle,environmentTitle,listener);
-                    runItem.failure = keyframes;
+					/* runItem.failure = keyframes; */
```

Comment:
so failed information is not needed anymore?

Reply:
Yeah, all keyframe messages are missing. That's one of the most important places in the plugin

Reply:
removed comments, added code into repo